### PR TITLE
DCOS-40668: Fix create component template in UI kit

### PR DESCRIPTION
**Pull request by weblancaster**
Fix component template paths

Closes DCOS-40668

## How to test

1. `npm run create:component blabla`
2. Go to `packages/index.ts` and export your module as so `export { Blabla } from "./blabla";`
3. `npm start` then visit `http://localhost:6006/`
4. Make sure you can see the component `blabla` on the left menu

### CODE REVIEW

File: scripts/templates/component/stories/Component.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
 import { withReadme } from "storybook-readme";
-import ${Component} from "../${Component}";
+import { ${Component} }from "../../index";
```
Comment (nLight) [inline scripts/templates/component/stories/Component.stories.tsx]: Missing space before `from` and extra `../`
should be `import { ${Component} } from "../index";`

File: scripts/templates/component/tests/Component.test.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import ${Component} from ".";
+import { ${Component} }from "../";
```
Comment (nLight) [inline scripts/templates/component/tests/Component.test.tsx]: missing space before `from`

**Comment by nLight** [comment]
Would appreciate some testing guides. What I did:

1. `npm run create:component Dima`
2. Check if everything looks ok

**Comment by GeorgiSTodorov** [comment]
I also created a new component to see if it works and checked the paths that were changed.

**Comment by mperrotti** [comment]
I ran the same thing Dima did (found [here](https://github.com/dcos-labs/ui-kit/blob/master/CONTRIBUTING.md#creating-a-new-component)) and saw the same issues he did. Should be good to go after fixing those typos.

File: scripts/templates/component/stories/Component.stories.tsx (hunk 1 above)
Comment (weblancaster) [reply scripts/templates/component/stories/Component.stories.tsx]: Ah good catch 😞

File: scripts/templates/component/tests/Component.test.tsx (hunk 2 above)
Comment (weblancaster) [reply scripts/templates/component/tests/Component.test.tsx]: Ah good catch 😞

File: scripts/templates/component/stories/Component.stories.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
 import { withReadme } from "storybook-readme";
-import ${Component} from "../${Component}";
+import { ${Component} } from "../../index";
```
Comment (nLight) [inline scripts/templates/component/stories/Component.stories.tsx]: Should be `"../index"`

Comment (weblancaster) [reply scripts/templates/component/stories/Component.stories.tsx]: Why should be `../index`? the reason we are using `../../index` is to target the root path as the projects will.

Comment (nLight) [reply scripts/templates/component/stories/Component.stories.tsx]: Ha! I get it now, so after scaffolding one should put the component into the main index

Comment (weblancaster) [reply scripts/templates/component/stories/Component.stories.tsx]: Correct. I added that to the description too.

Thanks.

**Pull request merged**